source/_integrations/emoncms.markdown

```diff
@@ -21,17 +21,54 @@ sensor:
   id: 1
 ```
 
+As of Feb 2020 including the option `use_emoncms_unit` will discover all sensors from Emoncms and use the unit of measurement specified in the Feed in Emoncms. Tested with [Emoncms](https://github.com/emoncms/emoncms) V10.1.13.
+
 ## Configuration variables
 
-- `api_key` (*Required*): The read API key for your Emoncms user.
-- `url` (*Required*): The base URL of Emoncms, use <https://emoncms.org> for the cloud based version.
-- `id` (*Required*): Positive integer identifier for the sensor. Must be unique if you specify multiple Emoncms sensors.
-- `include_only_feed_id` (*Optional*): Positive integer list of Emoncms feed IDs. Only the feeds with feed IDs specified here will be displayed. Can not be specified if `exclude_feed_id` is specified.
-- `exclude_feed_id` (*Optional*): Positive integer list of Emoncms feed IDs. All the feeds will be displayed as sensors except the ones listed here. Can not be specified if `include_only_feed_id` is specified.
-- `sensor_names` (*Optional*): Dictionary of names for the sensors created that are created based on feed ID. The dictionary consists of `feedid: name` pairs. Sensors for feeds with their feed ID mentioned here will get the chosen name instead of the default name
-- `value_template` (*Optional*): Defines a [template](/docs/configuration/templating/#processing-incoming-data) to alter the feed value.
-- `scan_interval` (*Optional*): Defines the update interval of the sensor in seconds.
-- `unit_of_measurement` (*Optional*): Defines the unit of measurement of for all the sensors. default is "W".
+{% configuration %}
+api_key:
+  description: The read API key for your Emoncms user.
+  required: true
+  type: string
+url:
+  description: "The base URL of Emoncms, use <https://emoncms.org> for the cloud based version. For self hosted Emoncms or EmonPi you may need a URL of <http://x.x.x.x/emoncms>."
```

Comment:
The problem with textlint lies here, in line 34

Reply:
But the lower case versions are in a URL.

Reply:
The problem here is quite simple, the last URL you made, isn't a valid URL.
Hence, not handled as one.

Reply:
How would you suggest identifying the need for a private/LAN IP here then?

<http://<YOUR_IP>/emoncms>